### PR TITLE
feature/TASK-4_shortlinker-api

### PR DESCRIPTION
**Pull request by JeriRov**

### CODE REVIEW

File: 04_shortlinker_api/src/controllers/url.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import {Request, Response, NextFunction} from 'express';
+
+export const shortUrl = async (req: Request, res: Response, next: NextFunction) => {
+    //TODO: Implement
+}
```
Comment (milkanuch) [inline 04_shortlinker_api/src/controllers/url.ts]: Do you really need to commit this?

File: 04_shortlinker_api/src/services/url.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export {};
```
Comment (milkanuch) [inline 04_shortlinker_api/src/services/url.ts]: empty export

File: 04_shortlinker_api/src/controllers/url.ts (hunk 1 above)
Comment (JeriRov) [reply 04_shortlinker_api/src/controllers/url.ts]: Don't worry, this is still a WIP version. I'm working on it.

File: 04_shortlinker_api/src/services/url.ts (hunk 2 above)
Comment (JeriRov) [reply 04_shortlinker_api/src/services/url.ts]: I am also working on this.

**Pull request merged**